CMakeLists.txt

```diff
@@ -67,31 +66,23 @@ ENDIF(LIBNFC_CONFFILES_MODE)
 option (BUILD_EXAMPLES "build examples ON/OFF" ON)
 option (BUILD_UTILS "build utils ON/OFF" ON)
 
-option (BUILD_DEBPKG "build debian package ON/OFF" OFF)
+if(MSVC)
```

Comment:
What is the reason for the difference in default value between platforms?

Reply:
The installation of man-pages is not necessary on Windows, as it doesn't support use of man-pages.

Reply:
Expect MSYS2 and Cygwin